helm-charts/falcon-sensor/templates/clusterrole.yaml

```diff
@@ -35,7 +35,9 @@ rules:
   - securitycontextconstraints
   resourceNames:
 {{- if .Values.node.enabled }}
-  - {{ include "falcon-sensor.fullname" . }}-node
+  - privileged
+  verbs:
+  - use
```

Comment:
When I tested this out, I got some double verbs:

```
rules:
- apiGroups:
  - ""
  - image.openshift.io
  resources:
  - imagestreams/layers
  verbs:
  - get
- apiGroups:
  - security.openshift.io
  resources:
  - securitycontextconstraints
  resourceNames:
  - privileged
  verbs:
  - use
  verbs:
  - use
```

a "dry-run=server" apply didn't fail though

Reply:
`helm template --validate --values=ci/cid-values.yaml --set node.enabled=true .`